mods/cnc/rules/defaults.yaml

```diff
@@ -491,7 +491,7 @@
 	Health:
 		HP: 400
 	Tooltip:
-		Name: Civilian Building
+		GenericName: Civilian Building
 		GenericVisibility: None
```

Comment:
Fixed, but I think I am suffering from yaml merge problems elsewhere. Also tooltips a la "Enemy Civilian Building" isn't really worth it.